dockstore-webservice/src/main/java/io/dockstore/webservice/core/Event.java

```diff
@@ -118,6 +124,7 @@ public class Event {
 
     public Event() { }
 
+    /*
```

Comment:
Hmm, big comment block. Should this constructor be deleted?

Reply:
Oh, yeah, I think so, this was part of an experiment that I forgot to reverse...

Reply:
Removed the [commented-out] big constructor.